sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -774,4 +774,23 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
       }
     }
   }
+
+  test("SPARK-25717: Insert overwrite a recreated external and partitioned table "
```

Comment:
Can you make the title shorter in a single line?

Reply:
```SPARK-25717: Insert overwrite a recreated external table should remove old-part first```

Would it be ok?

Reply:
How about `SPARK-25717: Insert overwrites remove old partition dirs correctly`?

Reply:
Ok， thank you.